apps/portal/controllers/apis/login.jag

```diff
@@ -96,13 +102,13 @@ var user;
             }
             else {
                 if (dest) {
-                    response.sendRedirect(dest + "login?loginAttempt=true");
+                    response.sendRedirect('/portal/login?loginAttempt=true');
                 }
             }
         }
         else {
             if (dest) {
```

Comment:
No need to do this check, 'if (dest) {}', as we are anyway sending user to '/portal/login?loginAttempt=true'